build.sbt

```diff
@@ -134,101 +66,70 @@ def scalaVersionSpecificFolders(srcName: String, srcBaseDir: java.io.File, scala
     List(CrossType.Pure, CrossType.Full)
       .flatMap(_.sharedSrcDir(srcBaseDir, srcName).toList.map(f => file(f.getPath + suffix)))
   CrossVersion.partialVersion(scalaVersion) match {
-    case Some((2, y))     => extraDirs("-2.x") ++ (if (y >= 13) extraDirs("-2.13+") else Nil)
-    case Some((0 | 3, _)) => extraDirs("-2.13+") ++ extraDirs("-3.x")
+    case Some((2, y))     => if (y >= 13) extraDirs("-2.13+") else Nil
+    case Some((0 | 3, _)) => extraDirs("-2.13+")
     case _                => Nil
   }
 }
 
-ThisBuild / mimaFailOnNoPrevious := false
-
-def doctestGenTestsDottyCompat(isDotty: Boolean, genTests: Seq[File]): Seq[File] =
-  if (isDotty) Nil else genTests
-
 lazy val commonSettings = Seq(
-  scalacOptions ++= commonScalacOptions(scalaVersion.value, isScala3.value),
   Compile / unmanagedSourceDirectories ++= scalaVersionSpecificFolders("main", baseDirectory.value, scalaVersion.value),
   Test / unmanagedSourceDirectories ++= scalaVersionSpecificFolders("test", baseDirectory.value, scalaVersion.value),
-  resolvers ++= Seq(Resolver.sonatypeRepo("releases"), Resolver.sonatypeRepo("snapshots")),
-  Test / parallelExecution := false,
-  testFrameworks += new TestFramework("munit.Framework"),
+  Compile / packageSrc / mappings ++= {
+    val base = (Compile / sourceManaged).value
+    (Compile / managedSources).value.map { file =>
+      file -> file.relativeTo(base).get.getPath
+    }
+  },
```

Comment:
I'd like to upstream this in https://github.com/typelevel/sbt-typelevel/issues/239.